### PR TITLE
feat(UI): Escort HUD improvements

### PR DESCRIPTION
**Pull request by TomGoodIdea**
## Feature Details
1. Disabled ships are displayed in grey to avoid confusing them with docked ships.
2. Escort selection is indicated with a pointer instead of icon color change.

## UI Screenshots
Before:

After:

(Both images show the same situation: the Black Diamonds carried and selected, the Shield Beetle disabled, the first Albatross selected, and the second Albatross disabled in another system.)

## Performance Impact
N/A

## Alternative Approaches
#4711 suggested to use the selection color for icons more frequently, but I think using a pointer is better as it doesn't hide normal icon colors.

### CODE REVIEW

**Comment by Saugia** [comment]
I'm not sure about having a phrase clutter the UI there more, as opposed to something with the icon.

For example, maybe using a grey for docked, as opposed to the purple of being disabled.

Or, perhaps a small indicator. Maybe something here. 
image

I'm open to discussion though.

**Comment by TomGoodIdea** [comment]
I'd rather change the disabled color to be consistent with other parts of the UI.
Out-of-system and docked ships are similar as both are "not here", and I'd leave them purple.

**Comment by Zireael07** [comment]
Disabled color may be made consistent but disabled and out of system should be somehow differentiated.

**Comment by TomGoodIdea** [comment]
Do you mean displaying disabled ships with the disabled color both in-system and out-of-system?

**Comment by Zireael07** [comment]
Disabled ships should be a single color in both, with out of system indicated via an icon/letter IMHO.

**Comment by TomGoodIdea** [comment]
Out-of-system ships are already indicated by their system name.

**Comment by Zireael07** [comment]
Ah. In that case, belay that

**Comment by Zitchas** [comment]
The more our information is primarily conveyed via icons or text, with colors being merely a secondary or alternative means of conveying the information, the better. This is more accessible, more universal, and more precise.

**Comment by warp-core** [comment]
> The more our information is primarily conveyed via icons or text, with colors being merely a secondary or alternative means of conveying the information, the better. This is more accessible, more universal, and more precise.

Icons or text are not necessarily any of those things.

**Comment by Zitchas** [comment]
"Hostile" is very specific word. "a color that is kind of reddish and in some circumstances can be confused with a thousand other colors" *might* mean hostile, or it might mean a whole variety of other things, and to some people it may be a completely different color.

Of course, it's all in how it's used, and legends, and whatnot. But overall, "hostile" is more directly meaningful to everyone than "kind-of-reddish-color".

In any case, text and and icons have a greater capacity for accuracy, precision, and universality.

**Comment by TomGoodIdea** [comment]
But, as Saugia noted, when we'd use too much text, the UI could become too cluttered.

And icons aren't necessarily universal and easily readable, especially if they're too small.

**Comment by Zitchas** [comment]
Oh, definitely. They need to be used carefully, with a good balance between readability, quantity, etc. And definitely need documentation and tooltips and stuff. Far too much of our game isn't documented for players. But they still beats a color that a not insignificant number of people don't even see the same. 

In terms of conveying meaning, color ranks pretty much at the bottom of the list, above only such things as "text too small to be legible" and "meaninglessly generic word/symbol" and "arcane symbol that isn't explained anywhere and doesn't resemble anything."

In other words, yes, words and symbols can definitely be done badly. But even a mediocre word/symbol is a strong improvement over pretty much all uses of color.
